src/lib.rs

```diff
@@ -306,6 +325,13 @@ where
         let bufs = &mut *guard;
         let mut event_buf = &mut bufs.current_buf;
 
+        // Time.
+
+        // TODO(TmLev): Error handling?
+        self.timer
+            .format_time(&mut event_buf)
+            .expect("Unable to write time to buffer");
```

Comment:
I'm really not sure how to handle errors here.

Reply:
This is perfectly alright. The only way these panics can occur is if we can't actually write to our buffer, which can only really happen when we run out of memory.

Reply:
Removed `TODO`.